src/components/FeatureModel/FeatureModelTree.vue

```diff
@@ -514,7 +545,7 @@ export default {
 
 #svg-container {
     width: 100%;
-    height: calc(100vh - 64px);
+    height: 100%;
```

Comment:
Changed this because i always had a weird scrollpane after putting the info, constraints, legend buttons in a v-row. When using the 100% everything seems to work fine.